services/QuillLMS/app/models/old_concept_result.rb

```diff
@@ -40,4 +40,27 @@ def correct?
   def concept_uid=(concept_uid)
     self.concept = Concept.where(uid: concept_uid).first
   end
+
+  # Over-riding the `bulk_insert` classmethod from the bulk_insert gem because the
+  # behavior documented in the gem doesn't actually work.  This change makes it behave
+  # as documented when passed `return_primary_keys: true`.  Minimal modifications from
+  # the original code found here, specifically around return values in if blocks:
+  # https://github.com/jamis/bulk_insert/blob/master/lib/bulk_insert.rb
+  def self.bulk_insert(*columns, values: nil, set_size: 500, ignore: false, update_duplicates: false, return_primary_keys: false)
```

Comment:
Maybe we should use a different method name here, rather than overwriting the gem's api (this could be surprising to someone later)?

Reply:
Yeah, that seems super-reasonable.  I'll change it to `custom_bulk_insert` unless you have a something else you prefer.